ssdutil/main.py

```diff
@@ -6,20 +6,49 @@
 #
 
 try:
-    import argparse
     import os
     import sys
+    import argparse
+    import json
+    import psutil
+    from blkinfo import BlkDiskInfo
 
-    from sonic_py_common import device_info, logger
+    from sonic_py_common import device_info, syslogger
 except ImportError as e:
     raise ImportError("%s - required module not found" % str(e))
 
 DEFAULT_DEVICE="/dev/sda"
 SYSLOG_IDENTIFIER = "ssdutil"
+DISK_TYPE_SSD = "sata"
 
 # Global logger instance
-log = logger.Logger(SYSLOG_IDENTIFIER)
+log = syslogger.SysLogger(SYSLOG_IDENTIFIER)
+
+def get_default_disk():
+    """Check default disk"""
+    default_device = DEFAULT_DEVICE
+    host_mnt = '/host'
+    host_partition = None
+    partitions = psutil.disk_partitions()
+
+    for parts in partitions:
+        if parts.mountpoint == "/host":
+            host_partition = parts
+            break
 
+    disk_major = os.major(os.stat(host_partition.device).st_rdev)
+    filters = {
+       'maj:min': '{}:0'.format(disk_major)
+    }
+
+    myblkd = BlkDiskInfo()
+    my_filtered_disks = myblkd.get_disks(filters)
```

Comment:
Good to check for 'my_filtered_disks' is valid before proceeding further

Reply:
Fixed in latest.